app/src/main/res/layout/fragment_list_item.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="utf-8"?>
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent"
+    android:layout_height="64dp">
+
+    <LinearLayout
```

Comment:
Why do you need this `LinearLayout` if it has `ConstraintLayout` as a parent? Use one of them.

Reply:
I could not do it using only "ConstraintLayout" or only "LinearLayout", the problem is that when there is no "description" I had to have "title" in the middle, and when there was, they should be exactly in the middle. In what I did, they divide the free space on a level. Are any crashs possible in this case?

Reply:
No, the problem is not with possible crash. It is just inefficient usage of layouts, basically both `LinearLayout` and `ConstraintLayout` are viewgroups (which holds other views), there is not much value on nesting them. `ConstraintLayout` is much more flexible, you should be able to achieve the behavior you want. Try to look at a few `ConstraintLayout` tutorials, or stackoverflow.